Restructure TOC and add get started (#276)
grafana/writers-toolkit: docs/sources/structure/_index.md

**Top-level:** A table of contents top-level represents groups of features and functions of a product. The first step in contributing to the docs is to identify which top-level entity you will be contributing to.

{{% admonition type="note" %}}

> You should not add a top-level entity to the table of contents. Reach out to the technical documentation team if it is not clear where your documentation belongs.

Reviewer: ```suggestion
Do not add a top-level entity to the table of contents. 
If you are not sure where your documentation belongs, contact the technical documentation team.
```
